recipes/conda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
+  number: 0
+  script: CONDA_DEFAULT_ENV='' python setup.py install
+
+requirements:
+  build:
+    - python
+    - conda-env
+    - pycosat
+    - pyyaml
+    - requests
+
+  run:
+    - python
+    - conda-env
+    - pycosat
+    - pyyaml
+    - requests
+
+test:
+  imports:
+    - conda
+
+  commands:
+    - which conda  # [unix]
+    - where conda  # [win]
```

Comment:
Same story as above. I believe I had trouble running these commands. Maybe it tries to use the `root` copy of `conda`. Hopefully fixing the links helps. 😕